examples/example-generation/types/typelist

```diff
@@ -0,0 +1,873 @@
+ObjectDetector
```

Comment:
That's perfect but it would be better if that file is generated on each build so that we don't have to change it manually